README.md

```diff
@@ -105,8 +105,8 @@ The following is a tiny "Hello World"-like example that you can run to have a su
 
 ```python
 import json
+import geojson
 import xyzspaces as xyz
```

Comment:
shall we change this to 
```python
from xyzspaces import XYZ
```
and below line as 
```python
xyz = XYZ(credentials="MY_XYZ_TOKEN")
```

Reply:
We could. I wanted to have imports without "from"...